paper/paper.md

```diff
@@ -0,0 +1,210 @@
+(with one example model shown in \autoref{fig:garnetsolution}),
+including methods to convert from one endmember basis to another
+[@Myhill:2021]. A composite material model
+allows calculation of the properties of assemblages of several phases
+and includes several seismic averaging schemes.
+
+![Properties of pyrope-grossular garnet at 1 GPa according to a published
+model [@Jennings:2015], as output by `BurnMan`.
+The excess Gibbs energy is useful for calculating phase equilibria by
+Gibbs minimization, while the endmember activities can be used to
+determine equilibrium via the equilibrium relations [@Holland:1998].
+\label{fig:garnetsolution}](figures/mg_ca_gt_properties.png)
+
+`BurnMan` includes many higher level functions that build on
+its material model classes. These include functions to
+fit thermodynamic models for both endmembers
+and solutions including full error propagation (\autoref{fig:fit}),
+construction of self-consistent planetary models
+and the calculation of seismic properties. An example of the
+construction of planetary models in `BurnMan`
+is shown in \autoref{fig:zog}.
+
+![Optimized fit of a PV equation of state [@Holland:2011] to
+stishovite data [@Andrault:2003], including 95% confidence intervals
+on both the volume and the bulk modulus.
+\label{fig:fit}](figures/stishovite_fitting.png)
+
+![A 1D profile through Planet Zog, a planet much like Earth, with an
+inner and outer core (blue and orange layers),
+isentropic convecting lower and upper mantle (green and red), and a
+depleted lithosphere (lilac) split into mantle and crust.
+Zog has the same mass (5.972e+24 kg) and moment of inertia factor
+(0.3307) as Earth. `BurnMan` ensures that the gravity and pressure
+profiles satisfy hydrostatic equilibrium, and allows different layers
+to have different thermal profiles, including an isentropic profile
+with thermal boundary layers
+(shown here for the upper and lower mantles and for the core).
+The computed geotherm is compared to several from the literature
+[@Stacey:1977;@Brown:1981;@Anderson:1982;@Alfe:2007;@Anzellini:2013].
+\label{fig:zog}](figures/zog.png)
+
+`BurnMan` also provides many utility classes and functions, including
+those required to undertake bulk and phase chemical calculations,
+to aid preparation and analysis of experiments.
+An example that uses bulk and phase compositions from samples
+to estimate phase proportions is shown in \autoref{fig:mars_fit}.
+
+![Mineral phase proportions in the mantle of Mars, estimated from
+high pressure experimental data [@Bertka:1997]. Weighted residuals
+(misfits) are also shown, indicating that the majority of experimental
+run products are consistent with the reported starting composition.
+\label{fig:mars_fit}](figures/fit_mars_experiments.png)
+
+`BurnMan` does not attempt to replicate Gibbs minimization codes,
+of which there are many (PerpleX, HeFESTo, TheriakDomino, FactSAGE and MageMin).
```

Comment:
I think we need references for all codes we list by name. The point of JOSS is to improve the citation of codes and reviewers will likely be picky about this.

Reply:
Yes, definitely, thanks for the nudge.